src/lib/decorations/brace.typ

```diff
@@ -41,11 +43,13 @@
 ///   #show-parameter-block("amplitude", ("number"), [
 ///     Sets the height of the brace, from its baseline to its middle tip.], default: .5)
 ///   #show-parameter-block("pointiness", ("ratio", "angle"), [
-///     How pointy the spike should be. #0deg or `0%` for maximum pointiness, #90deg or `100%` for minimum.], default: 15deg)
+///     How pointy the spike should be. #0deg or `100%` for maximum pointiness, #90deg or `100%` for minimum.], default: 15deg)
 ///   #show-parameter-block("outer-pointiness", ("ratio", "angle"), [
-///     How pointy the outer edges should be. #0deg or `0` for maximum pointiness (allowing for a smooth transition to a straight line), #90deg or `1` for minimum. Setting this to #auto will use the value set for `pointiness`.], default: 15deg)
-///   #show-parameter-block("content-offset", ("number","length"), [
+///     How pointy the outer edges should be. #0deg or `100%` for maximum pointiness (allowing for a smooth transition to a straight line), #90deg or `1` for minimum. Setting this to #auto will use the value set for `pointiness`.], default: 15deg)
```

Comment:
Here is a similar change that makes little sense to me: 100% = maximum, 1 = minimum